upup/models/cloudup/resources/addons/networking.flannel/k8s-1.6.yaml

```diff
@@ -1,3 +1,45 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: flannel
+  labels:
+    role.kubernetes.io/networking: "1"
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+    verbs:
+      - get
+  - apiGroups:
+      - ""
+    resources:
+      - nodes
+    verbs:
+      - list
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - nodes/status
```

Comment:
giving this power to a service account of a pod on every node means that every node has access to a credential that lets them change the status and labels of any other node. if you plan to make use of the NodeRestriction admission plugin added in 1.7 to limits nodes to modifying their own status, this undermines that limitation.